<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>375</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>375</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>290</width>
    <height>360</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>AoC Chat Exploit</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color: rgb(150,150,150); /*line*/
font: 8pt &quot;Fixedsys&quot;;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="Main">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <widget class="QLabel" name="lbPlayerName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	 color: rgb(0,255,0);
     background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Player Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbIpAddress">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>42</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	 color: rgb(0,255,0);
     background: transparent;
}</string>
    </property>
    <property name="text">
     <string>IP Address:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bGames">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>282</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	 color: rgb(0,190,0);
	 border-radius: 6px;
     background-color: black;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
 }
QPushButton::hover {
     background-color: rgb(10,10,10);
	 color: rgb(0, 255, 0);
 }</string>
    </property>
    <property name="text">
     <string>Show games</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bJoin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	 color: rgb(0,190,0);
	 border-radius: 6px;
     background-color: black;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
 }
QPushButton::hover {
     background-color: rgb(10,10,10);
	 color: rgb(0, 255, 0);
 }</string>
    </property>
    <property name="text">
     <string>Join!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu {
	 color: rgb(0,190,0);
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
}
QMenu::item::selected {     
     background-color: rgb(10,10,10);
	 color: rgb(0, 255, 0);
}
QLineEdit {
	 color: rgb(0,190,0);
	 text-align: center;
	 border-radius: 6px;
     background-color: black;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
 }
QLineEdit::hover {
     background-color: rgb(5, 5, 5);
	 color: rgb(0, 255, 0);
 }</string>
    </property>
    <property name="text">
     <string>H-BlacK</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>42</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMenu {
	 color: rgb(0,190,0);
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
}
QMenu::item::selected {     
     background-color: rgb(10,10,10);
	 color: rgb(0, 255, 0);
}
QLineEdit {
	 color: rgb(0,190,0);
	 text-align: center;
	 border-radius: 6px;
     background-color: black;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
 }
QLineEdit::hover {
     background-color: rgb(5, 5, 5);
	 color: rgb(0, 255, 0);
 }</string>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="LvGames">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>250</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView {
     color: rgb(0,190,0);
	 border-radius: 6px;
     background-color: black;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
}</string>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>360</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	 color: rgb(0,255,0);
     background: transparent;
     border-style: outset;
     border-width: 1px;
     border-color: rgb(150,150,150);
}</string>
    </property>
    <property name="text">
     <string>Status:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>63</y>
      <width>250</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Line {
background: transparent;
color: rgb(150,150,150);
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatusMsg">
    <property name="geometry">
     <rect>
      <x>62</x>
      <y>360</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	 color: rgb(0,255,0);
     background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Application loaded. hihihi!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>inicializechat()</slot>
 </slots>
</ui>
